recipes/shs-libcxi/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+context:
+  name: "shs-libcxi"
+  version: 12.0.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/HewlettPackard/shs-libcxi/archive/refs/tags/release/shs-${{ version }}.tar.gz
+    sha256: 9419292ffe08dd844360778b64de3e24c87206cf24c81cb1fb4607d3c10f6901
+    patches:
+      - fix-random.patch
+      - fix-udp-header.patch
+
+build:
+  number: 0
+  script: build.sh
+  skip:
+    - win
+    - osx
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - autoconf
+    - automake
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - libconfig
+    - libfuse ==2.9.9
+    - libnl
+    - libnuma
+    - libsystemd0
+    - libuv
+    - lm-sensors
+    - shs-cassini-headers
+    - shs-cxi-drivers
+    - yaml
+  run:
+    - libconfig
+    - libfuse ==2.9.9
```

Comment:
Please remove these pins

Reply:
I am afraid it is not compatible with libfuse3, so this is the latest version that can be used

Reply:
Replaced it by <3.0